content/post/glue-work.md

```diff
@@ -0,0 +1,83 @@
+
+But it also matters a lot because the distribution of glue work is not equitable. Women are more likely to [volunteer](https://hbr.org/2018/07/why-women-volunteer-for-tasks-that-dont-lead-to-promotions) for non-promotable tasks at work (much like they do at [home](https://www.bls.gov/tus/charts/household.htm), but that is another story). [Representation](https://drive.google.com/file/d/1gB74e9bnlFycOwnfFI2GgCM0uuWnN9Tr/view?usp=sharing) of women in analytics varies across specialties - women and men are close to equally represented among analysts, but less than a quarter of data scientists and engineers are women. Only 24% of analytics leaders are women. Tackling glue work is a real, tangible way we can help women advance and to work toward more equitable gender representation in analytics leadership.
+
+If you don’t have any women on your analytics team, well, [think about that](https://www.forbes.com/sites/metabrown/2017/03/30/she-asked-for-thoughts-on-women-in-analytics-heres-what-i-said/#1b4c7e35a5df) the next time you’re hiring. But in male-only groups, the same men tend to [volunteer](https://hbr.org/2018/07/why-women-volunteer-for-tasks-that-dont-lead-to-promotions) time after time for non-promotable tasks. So regardless of the gender makeup of your team, a small number of people on your team are probably doing this work without being rewarded for it.
+
+### The Meta-Story
+Arguably, analytics itself is glue within many companies. In most companies, the analytics team is not responsible for a revenue target. The team may not own any OKRs. The impact of analytics is subtler. Good analysts are multipliers, enabling the company to identify and focus on areas where changes can have the most impact. We don’t build features, we don’t close deals, but we shape the processes, tools, and cultures that enable companies to be smarter and faster across the board.
+
+Yet, there is often confusion about [what analysts are](https://hbr.org/2018/12/what-great-data-analysts-do-and-why-every-organization-needs-them) - they just write reports, right? Data science has the opposite problem - folks on the outside think it’s really exciting, but the reality is that it’s exciting occasionally, and boring a lot. In either role, when you tell someone that the “simple” request they asked for is really multiple days or weeks of work because the data isn't yet available, isn't well-defined, or needs significant transformation work, their will eyes glaze over.
```

Comment:
"their will eyes glaze over" --> "their eyes will glaze over"

However, you might want to say "may" instead of will, as it is not always the case!